### PR TITLE
Added Test Files For GraphQL Package

### PR DESCRIPTION
**Pull request by sharadregoti**

### CODE REVIEW

**Comment by sharadregoti** [comment]
Task linked: [CU-33qw99 Graphql Package Unit Testing](https://app.clickup.com/t/33qw99)

File: gateway/model/types.go [hunk 1]
```diff
@@ -8,6 +8,39 @@ import (
...
+// GraphQLFunctionInterface is an interface consisting of functions of function module used by graphql module
+type GraphQLFunctionInterface interface {
+	CallWithContext(ctx context.Context, service, function, token string, auth, params interface{}) (interface{}, error)
+}
+
+// GraphQLSchemaInterface is an interface consisting of functions of schema module used by graphql module
+type GraphQLSchemaInterface interface {
+	GetSchema(dbAlias, col string) (Fields, bool)
+}
+
```
Comment (YourTechBud) [inline gateway/model/types.go:43]: Create these in the `graphql` package. Put it in `utils/graphql/types.go`

File: gateway/modules/auth/handle_functions_test.go [hunk 2]
```diff
@@ -124,6 +124,7 @@ func TestIsFuncCallAuthorised(t *testing.T) {
 				t.Error("Got Error-", err, "Want Error-", test.IsErrExpected)
 			}
 			// check result if TokenClaims is returned after parsing token and matching rule
+
```
Comment (YourTechBud) [inline gateway/modules/auth/handle_functions_test.go:127]: Remove space

File: gateway/utils/constants.go [hunk 3]
```diff
@@ -5,7 +5,7 @@ import (
 )
 
 // BuildVersion is the current version of Space Cloud
-const BuildVersion = "0.18.2"
+const BuildVersion = "0.18.0"
```
Comment (YourTechBud) [inline gateway/utils/constants.go:8]:
```suggestion
const BuildVersion = "0.18.2"
```

File: gateway/utils/graphql/func.go [hunk 4]
```diff
@@ -26,6 +27,8 @@ func (graph *Module) execFuncCall(ctx context.Context, token string, field *ast.
 		return
 	}
 
+	log.Println("func name", funcName, timeout)
```
Comment (YourTechBud) [inline gateway/utils/graphql/func.go:30]: Remove this

File: gateway/utils/graphql/graphql.go [hunk 5]
```diff
@@ -203,6 +199,9 @@ func (graph *Module) execGraphQLDocument(ctx context.Context, node ast.Node, tok
 
 		if schema != nil {
 			fieldStruct, p := schema[field.Name.Value]
+			if p {
+				log.Println("Hello Graph", p, field.Name.Value, fieldStruct.IsLinked)
+			}
```
Comment (YourTechBud) [inline gateway/utils/graphql/graphql.go:204]: Remove this

**Pull request merged**